corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -521,9 +521,7 @@ <h3 class="caption" data-bind="html: header" tabindex="0"></h3>
                 }
             "/>
         <!-- /ko -->
-        <!-- ko ifnot: $parent.hideLabel -->
-          <span data-bind="renderMarkdown: $data"></span>
-        <!-- /ko -->
+        <span data-bind="renderMarkdown: $data, class: $parent.hideLabel ? 'sr-only' : ''"></span>
```

Comment:
does this "class" need to be in "attr" like above? I am not entirely sure how the syntax is to work here.

Reply:
Since we are requiring to define `sr-only` class based on binding value, it is not done here directly as an attribute.